tests/pyautomagic/test_Subject.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from pyautomagic.src.Subject import Subject
+
+
+def test_extract_name():
+    address = 'A\\B\\C\\sub-01'
```

Comment:
For consistency with the rest of the repo, use '/' instead of '\\' for this whole file. Should function the same way.